Form/Psr7RequestHandler.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PsrHttpMessage\Form;
+
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\UploadedFileInterface;
+use Symfony\Component\Form\Exception\UnexpectedTypeException;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\RequestHandlerInterface;
+
+/**
+ * Class Psr7RequestHandler.
+ */
+class Psr7RequestHandler implements RequestHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function handleRequest(FormInterface $form, $request = null): void
+    {
+        if (!$request instanceof Request) {
+            throw new UnexpectedTypeException($request, Request::class);
+        }
+
+        $name = $form->getName();
+        $method = $form->getConfig()->getMethod();
+
+        if ($method !== $request->getMethod()) {
+            return;
+        }
+
+        // For request methods that must not have a request body we fetch data
+        // from the query string. Otherwise we look for data in the request body.
+        if ($method === 'GET' || $method === 'HEAD' || $method === 'TRACE') {
```

Comment:
please use yoda conditions, especially here since this is a copy/paste from HttpFoundationRequestHandler, which uses yoda conditions here

Reply:
(same in all the files)